src/interfaces/tooltipNew.d.ts

```diff
@@ -3,11 +3,14 @@ import * as PopperJS from "popper.js";
 
 type RefHandler = (ref: HTMLElement | null) => void;
 
+export type TooltipTheme = "normal" | "invert" | "important";
+
 export type TooltipTriggerAction = "managed" | "click" | "hover";
 
 export interface IPopperTooltipProps
   extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
+  theme?: TooltipTheme;
```

Comment:
add the theme property to `ICssTooltipProps`

Reply:
ok

Reply:
still no `theme` in `ICssTooltipProps`

Reply:
line 53